### PR TITLE
Bug #13540: some improvements around the version tracking switching at component instance level

### PR DESCRIPTION
**Pull request by SilverYoCha**

### CODE REVIEW

File: core-war/src/main/webapp/attachment/jsp/displayAttachedFiles.jsp [hunk 1]
```diff
@@ -87,7 +88,9 @@
   <view:settings var="onlineEditingWithCustomProtocolAlert" settings="org.silverpeas.util.attachment.Attachment" defaultValue="${true}" key="attachment.onlineEditing.customProtocol.alert" />
   <c:set var="webdavEditingEnable" value="${mainSessionController.webDAVEditingEnabled && onlineEditingEnable}" />
```
Comment (mmoqui) [inline core-war/src/main/webapp/attachment/jsp/displayAttachedFiles.jsp:89]: Shouldn't the WebDAV editing be disabled if a huge processing is currently done on attachments?

File: core-war/src/main/webapp/attachment/jsp/editAttachedFiles.jsp [hunk 2]
```diff
@@ -38,13 +38,16 @@
 <%@ page import="org.silverpeas.core.contribution.attachment.model.SimpleDocument" %>
 <%@ page import="org.silverpeas.web.attachment.VersioningSessionController" %>
 <%@ page import="org.silverpeas.core.ResourceReference" %>
+<%@ page import="org.silverpeas.core.contribution.attachment.process.huge.AttachmentHugeProcessManager" %>
 <%@ include file="checkAttachment.jsp"%>
 <view:setConstant var="spinfire" constant="org.silverpeas.core.util.MimeTypes.SPINFIRE_MIME_TYPE" />
 <c:set var="mainSessionController" value="<%=m_MainSessionCtrl%>" />
 <view:settings var="onlineEditingEnable" settings="org.silverpeas.util.attachment.Attachment" defaultValue="${false}" key="OnlineEditingEnable" />
 <view:settings var="dAndDropEnable" settings="org.silverpeas.util.attachment.Attachment" defaultValue="${false}" key="DragAndDropEnable" />
 <c:set var="webdavEditingEnable" value="${mainSessionController.webDAVEditingEnabled && onlineEditingEnable}" />
```
Comment (mmoqui) [inline core-war/src/main/webapp/attachment/jsp/editAttachedFiles.jsp:47]: Shouldn't the WebDAV editing be disabled if a huge processing is currently done on attachments?

File: core-war/src/main/webapp/attachment/jsp/displayAttachedFiles.jsp (hunk 1 above)
Comment (SilverYoCha) [reply core-war/src/main/webapp/attachment/jsp/displayAttachedFiles.jsp:89]: The WebDAV is indeed disabled if a huge processing is currently done on attachments.
This variable is here given to the foolowing tag:
```
        <menu:simpleDocument attachment="${currentAttachment}"
                             showMenuNotif="${showMenuNotif}"
                             useWebDAV="${webdavEditingEnable}"
                             useXMLForm="${useXMLForm}"
                             fromAlias="${fromAlias}"
                             userRole="${highestUserRole}"/>
 ```
The webdav functionnalities are disabled from this tag.

Comment (mmoqui) [reply core-war/src/main/webapp/attachment/jsp/displayAttachedFiles.jsp:89]: ok

File: core-war/src/main/webapp/attachment/jsp/editAttachedFiles.jsp (hunk 2 above)
Comment (mmoqui) [reply core-war/src/main/webapp/attachment/jsp/editAttachedFiles.jsp:47]: I suppose it is disabled as it is explained in the previous comment

File: core-library/src/main/java/org/silverpeas/core/admin/component/model/LocalizedWarning.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+package org.silverpeas.core.admin.component.model;
+
+import org.silverpeas.core.ui.DisplayI18NHelper;
+
+import java.util.Map;
+
+/**
+ * @author ehugonnet
+ */
+public class LocalizedWarning {
```
[the remark this answers is not in the document]
Comment (SilverYoCha) [reply core-library/src/main/java/org/silverpeas/core/admin/component/model/LocalizedWarning.java:40]: Oki

File: core-war/src/main/webapp/attachment/jsp/editAttachedFiles.jsp (hunk 2 above)
Comment (SilverYoCha) [reply core-war/src/main/webapp/attachment/jsp/editAttachedFiles.jsp]: Yes, but I have updated the source anyway because of some missings.

**Pull request merged**